docs/inference.rst

```diff
@@ -354,17 +355,17 @@ Now run::
     # PSD estimation options
     PSD_ESTIMATION="H1:median L1:median"
     PSD_INVLEN=4
```

Comment:
Since you changed the PSD segment length and stride making them same as the search, I just checked that the psd inverse length we use in the search is 16.